db/seeds/development/publishers.seeds.rb

```diff
@@ -0,0 +1,12 @@
+logger = Logger.new(STDOUT)
```

Comment:
We don't need this and csv files. Just use facker to generate everything we need.